packages/oauth/src/utils/secure-storage.ts

```diff
@@ -0,0 +1,60 @@
+import { decrypt, encrypt, generateKey, toBase64, toByteArray } from './crypto'
+
+const SESSION_KEY_SALT = 'connect.salt'
```

Comment:
let's use a different name than the one from Connect plugin. if it's supposed to be used throughout Online, probably 'online.salt' is a better name.